gapis/resolve/report.go

```diff
@@ -117,10 +118,13 @@ func (r *ReportResolvable) Resolve(ctx context.Context) (interface{}, error) {
 		}
 	}
 
+	InitialCommands, _, _ := initialcmds.InitialCommands(ctx, r.Path.Capture)
```

Comment:
lowercase variable names
I'm not a fan of this new dependency. Especially since it looks like it's just here to undo the change in `find_issues.go`.

Reply:
No, it is not what is happening here. 

This has to be done because the issues coming from externs.go counted with the initial commands.

We have two different way to create issues and they have been counted differently before, we have to unify them in a single place. And this seems to be the best place for it,